templates/taxi/index.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% block title %}
+  <title>Home page</title>
```

Comment:
The `title` block should not include the `<title>` tags since the `base.html` already contains them. Instead, just provide the content for the title, like `Home page`.